### PR TITLE
Card #4

### PR DESCRIPTION
**Pull request by sayalis-cuelogic**
Designed template for adding and viewing comments using bootstrap.
Implemented add and view comment for particular blog.

### CODE REVIEW

File: commentpost/models.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from post.models import PostInfo
+# Create your models here.
+
+
+class PostCommentInfo(models.Model):
+    user_id = models.ForeignKey(User)
+    post_id = models.ForeignKey(PostInfo)
+    comment_data = models.CharField(max_length=200)
+    comment_date = models.DateTimeField(auto_now_add=True,
```
Comment (aloka-cuelogic) [inline commentpost/models.py]: blank=True and Null =True is not required with auto_add_now =True

File: commentpost/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+from commentpost import views
+
+urlpatterns = [
+    url(r'^(?P<post_id>[0-9]+)/add/$', views.AddCommentView.as_view(),
```
Comment (aloka-cuelogic) [inline commentpost/urls.py]: change this to add/id format

File: commentpost/views.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                "postdata": PostInfo.objects.all(),
+                "userdata": User.objects.all()}
+            return HttpResponseRedirect(reverse
+                                        ('comment:addcomment',
+                                         args=(post_id,)))
+        else:
+            return HttpResponseRedirect(reverse('comment:addcomment'),
+                                        {'form': form})
+
+    @csrf_exempt
```
Comment (aloka-cuelogic) [inline commentpost/views.py]: why csrf_exempt? 
PLease pass the csrf_token ?

**Comment by aloka-cuelogic** [comment]
@sayalis-cuelogic 
Please work on review comment

**Comment by sayalis-cuelogic** [comment]
Worked on review comments.
URL of add post page is changed as asked to change in review.

**Comment by aloka-cuelogic** [comment]
Reviewed The PR.
Working as expected.

**Pull request merged**